src/components/subsetDraft/SubsetMetadata.jsx

```diff
@@ -36,25 +36,23 @@ export const SubsetMetadata = ({subset}) => {
                 <div style={{float: 'left', marginRight: '20px', padding: '0'}}>
                     <label style={{display: 'block', fontSize: '16px', fontFamily: 'Roboto'}}
                            htmlFor="from_date">{t('Valid from')}: </label>
-                    <DatePicker id='from_date' style={{display: 'block'}}
-                                value={subset.draft.validFrom}
-                                onChange={(date) => subset.dispatch({action: 'from', data: date})}
-                                clearIcon={null}
-                                format='dd.MM.y'
-                                locale={languages.find(i => i.default).IETF}
+                    <input type="date" id='from_date' style={{display: 'block'}}
+                                value={subset.draft.validFrom?.toISOString().substr(0, 10)}
+                                onChange={(e) => {
+                                    const dateArr = e.target.value.split("-");
+                                    subset.dispatch({action: 'from', data: new Date(dateArr[0], dateArr[1], dateArr[2])});}}
                                 className='datepicker'/>
                 </div>
 
                 <div style={{float: 'left'}}>
                     <label style={{display: 'block', fontSize: '16px', fontFamily: 'Roboto'}}
                            htmlFor="to_date">{t('Valid to')}: </label>
```

Comment:
the Lint (Codacy) will say to put single quotes

Reply:
Actually Codacy is quiet for me! Maybe you have to configure it to be more angry about double quotes? I will change it, though.